Sources/StreamChat/Database/DTOs/PollDTO.swift

```diff
@@ -0,0 +1,177 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import CoreData
+import Foundation
+
+@objc(PollDTO)
+class PollDTO: NSManagedObject {
+    @NSManaged var allowAnswers: Bool
+    @NSManaged var allowUserSuggestedOptions: Bool
+    @NSManaged var answersCount: Int
+    @NSManaged var createdAt: DBDate
+    @NSManaged var pollDescription: String?
+    @NSManaged var enforceUniqueVote: Bool
+    @NSManaged var id: String
+    @NSManaged var name: String
+    @NSManaged var updatedAt: DBDate?
+    @NSManaged var voteCount: Int
+    @NSManaged var custom: Data?
+    @NSManaged var voteCountsByOption: [String: Int]?
+    @NSManaged var isClosed: Bool
+    @NSManaged var maxVotesAllowed: NSNumber?
```

Comment:
Int?

Reply:
Wasn't able to make it optional, it defaults to 0 if it's an Int in Core Data. And we need this one to be optional.